spec/BowlingGameSpec.js

```diff
@@ -0,0 +1,97 @@
+describe('bowling game logic', function(){
+  var game;
+
+  beforeEach(function(){
+    game = new BowlingGame();
+  });
+
+  it('starts with score = 0', function() {
+    expect(game.score).toEqual(0);
+  });
+
+  it('has 10 frames', function() {
```

Comment:
I can see why you would write this sort of test but it's not testing the public interface to game so you might want to delete it at some point.